### DISCUSSION TITLE
Forking does not work

### DISCUSSION THREAD

**Discussion by saaya-code** [community/community#47133] [Repositories] [answered]
Bug

Hello, there's an issue with forking, my and my colleagues are unable to fork a repository therefore we can't make pull requests.

**Comment by weidehai** [comment]
same here

**Comment by Bejte** [comment]
same here

**Comment by rakshaa-01** [comment]
Yeah, here's a screenshot of the bug
![Error](https://user-images.githubusercontent.com/97796804/218310239-3e5c633a-3d82-4c60-aec8-5186e79056bb.JPG)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @saaya-code ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by queenofcorgis** [comment, marked as answer]
This is now resolved - as we are not seeing these errors show up in our monitoring. Our monitors show this event happened February 12 between 6-8am ET. However, if you are still seeing this persist or have had this occur more recently, please let me know ❤️ .

**Reply by MSkhaliq** [reply]
Hi , 

I am unable to fork ,did not shown any error, can you please guide me ???

**Reply by OneOfTheMicahs** [reply]
I am not getting the unicorn "This page is taking too long to load" message, but I am unable to fork a repository. I created a discussion a few days ago before I found this thread: https://github.com/orgs/community/discussions/114482.

**Comment by traitsisgiorgos** [comment]
same!!

**Comment by therealsupermax** [comment]
I've just tried to fork a project I was assigned to and github won't complete the fork to my profile. What gives?

**Reply by dmore** [reply]
can you re-open the issue please if relevant.
